client/api/assessments.js

```diff
@@ -0,0 +1,12 @@
+import request from 'superagent'
+
+const URL = 'http:localhost:3000/v1/assessments'
```

Comment:
Have a look at what the api is actually called and make changes here

Reply:
Fixed spelling error in URL and resolved conflicts